src/nvim/os_unix_defs.h

```diff
@@ -145,7 +145,7 @@
 
 
 #ifndef USR_GVIMRC_FILE
-#  define USR_GVIMRC_FILE "$HOME/.ngvimrc"
+#  define USR_GVIMRC_FILE "~/.ngvimrc"
 #endif
 
 #ifndef USR_GVIMRC_FILE2
```

Comment:
Can SYS_GVIMRC_FILE be removed as well? Given that nvim will defer to external programs for GUIs, I don't think it's needed.